rdwatch/scoring/tasks/animation_export.py

```diff
@@ -0,0 +1,698 @@
+        else:  # 'mp4'
+            # Create a temporary file for MP4
+            temp_file = tempfile.NamedTemporaryFile(
+                prefix=prefix, suffix='.mp4', dir=output_dir, delete=False
+            )
+            with open(temp_file.name, 'w') as tmp_file:
+                output_file_path = tmp_file.name
+
+                # Initialize video writer
+                fourcc = cv2.VideoWriter_fourcc(*'mp4v')
+                video_width = max_width_px
+                video_height = max_height_px
+                if rescale:
+                    video_width = rescaled_max_width
+                    video_height = rescaled_max_height
+                video_writer = cv2.VideoWriter(
+                    output_file_path, fourcc, fps, (video_width, video_height)
+                )
+
+                # Write each frame to the video
+                for frame in np_array:
+                    frame_bgr = cv2.cvtColor(
+                        frame, cv2.COLOR_RGB2BGR
+                    )  # Convert RGB to BGR for OpenCV
+                    video_writer.write(frame_bgr)
+
+                video_writer.release()  # Close the video writer
+        name = f'{prefix}.{output_format}'
+        self.update_state(
+            state='SUCCESS',
+            meta={
+                'current': count,
+                'total': total_images,
+                'mode': 'Rendering Complete',
+                'siteEvalId': site_evaluation_id,
+            },
+        )
+
+        return output_file_path, name
+    return False, False
+
+
+@app.task(bind=True)
+def create_site_animation_export(
+    self, site_evaluation_id: UUID4, settings: GenerateAnimationSchema, userId: int
+):
+    task_id = self.request.id
+    try:
+        site_evaluation = Site.objects.get(pk=site_evaluation_id)
+    except Site.DoesNotExist:
+        site_evaluation = AnnotationProposalSite.object.get(pk=site_evaluation_id)
+    user = User.objects.get(pk=userId)
+    site_export = AnimationSiteExport(
+        name='',
+        created=datetime.now(),
+        site_evaluation=site_evaluation.uuid,
+        user=user,
+        celery_id=task_id,
+        arguments=settings,
+    )
+    # try:
+    site_export.celery_id = task_id
+    site_export.save()
+    file_path, name = create_animation(self, site_evaluation_id, settings)
+    if file_path is False and name is False:
+        logger.warning('No Images were found')
+        site_export.delete()
+        return
+    site_export.name = name
+    with open(file_path, 'rb') as file:
+        site_export.export_file.save(name, File(file))
+    site_export.save()
+    if os.path.exists(file_path):
+        os.remove(file_path)
+    # except Exception as e:
+    #     logger.warning(f'Error when processing Animation: {e}')
+    #     if site_export:
+    #         site_export.delete()
+
+
+@app.task(bind=True)
+def create_modelrun_animation_export(
+    self, modelrun_id: UUID4, settings: GenerateAnimationSchema, userId: int
```

Comment:
Similar to above issue -
```suggestion
    self, modelrun_id: UUID4, settings: dict[str,  Any], userId: int
```